### PR TITLE
New gms-artic version and change name of files for nanopore analyses

### PR DESCRIPTION
**Pull request by henningonsbring**
### The purpose of the code changes are as follows:
- New gms-artic version
- File name change at end of Nanopore analysis in same way as for Illumina

### Preparations:

**How to prepare mutant for test**:
* `cd MUTANT`
* `bash mutant/standalone/deploy_hasta_update.sh stage <MUTANT_branch>`

**How to prepare cg for test**:
- `us`
- Paxa and update cg:
  - `paxa -u <user> -s hasta -r cg-stage`
  - `bash update-cg-stage.sh master`
- If needed, paxa and update servers:
  - `paxa -u <user> -s hasta -r servers-stage`
  - `bash update-servers-stage.sh <master/branch>`
  
### How to test:
Run in a tmux screen or similar if testing on a large dataset.

**Test with cg**:
- `us`
- `cg workflow mutant start maturejay`

### Expected outcome:
- [ ] Produced files contain expected values

### Review:
- [ ] Code reviewed by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henningonsbring** [comment]
Mutant can now run without errors (started via cg), see end of output from `cg workflow mutant start truecoyote`:
```
Analysis successfully stored in Housekeeper: truecoyote : 2022-07-28 09:57:49
Storing analysis in StatusDB for truecoyote
Running command source activate S_mutant; /home/proj/bin/conda/envs/S_mutant/bin/mutant --version
Analysis successfully stored in StatusDB: truecoyote : 2022-07-28 09:57:49
Action None set for case truecoyote
```

**Comment by henningonsbring** [comment]
Result file contain what it should contain (besides that the header needs to be updated):
```
cat /home/proj/stage/mutant/cases/truecoyote/results/sars-cov-2_489423_results.csv 
Sample,Selection,Region Code,Ticket,%N_bases,%10X_coverage,QC_pass,Lineage,PangoLEARN_version,VOC,Mutations
22CS999901,Information saknas,01,489423,2.31,99.17,TRUE,AY.45,4.1.2,No,D614G;H655Y;P681R
22CS999902,Information saknas,01,489423,0.64,99.4,TRUE,AY.43,4.1.2,No,D614G
22CS999903,Information saknas,01,489423,4.44,97.39,TRUE,AY.5,4.1.2,No,D614G;P681R
22CS999904,Information saknas,01,489423,16.93,88.85,FALSE,AY.4,4.1.2,No,L452R;T478K;D614G;P681R
22CS999905,Information saknas,01,489423,99.8,0.0,FALSE,Unassigned,4.1.2,No,-
22CS999906,Information saknas,01,489423,2.41,98.03,TRUE,AY.43,4.1.2,No,L452R;D614G
22CS999907,Information saknas,01,489423,34.13,68.42,FALSE,B.1.617.2,4.1.2,VOC,L452R;T478K;D614G;P681R
22CS999908,Information saknas,01,489423,22.94,82.39,FALSE,B.1.617.2,4.1.2,VOC,G142D;L452R;T478K;D614G;P681R
22CS999909,Information saknas,01,489423,8.01,96.27,TRUE,AY.9.2,4.1.2,No,D614G
22CS999910,Information saknas,01,489423,29.98,77.75,FALSE,B.1.617.2,4.1.2,VOC,G142D;D614G;P681R
22CS999911,Information saknas,01,489423,14.32,90.63,TRUE,AY.122,4.1.2,No,D614G
22CS999912,Information saknas,01,489423,16.97,85.83,FALSE,AY.4,4.1.2,No,L452R;D614G;P681R
22CS999913,Information saknas,01,489423,8.45,92.39,TRUE,AY.42,4.1.2,No,D614G;P681R
22CS999914,Information saknas,01,489423,4.19,98.03,TRUE,AY.9.2,4.1.2,No,D614G
22CS999915,Information saknas,01,489423,5.85,95.81,TRUE,AY.9.2,4.1.2,No,D614G;P681R
22CS999916,Information saknas,01,489423,3.48,97.97,TRUE,AY.9.2,4.1.2,No,D614G
22CS999917,Information saknas,01,489423,12.78,94.65,TRUE,AY.9.2,4.1.2,No,A222V;L452R;T478K;D614G;P681R
22CS999918,Information saknas,01,489423,6.57,94.85,TRUE,AY.9.2,4.1.2,No,A222V;D614G;P681R
22CS999919,Information saknas,01,489423,3.54,98.94,TRUE,AY.9.2,4.1.2,No,Q613H;D614G
22CS999920,Information saknas,01,489423,5.87,95.75,TRUE,AY.9.2,4.1.2,No,G142D;A222V
22CS999921,Information saknas,01,489423,63.28,44.56,FALSE,B.1.617.2,4.1.2,VOC,L452R;T478K
22CS999922,Information saknas,01,489423,10.03,93.77,TRUE,AY.9.2,4.1.2,No,A222V
22CS999923,Information saknas,01,489423,7.98,97.47,TRUE,AY.122,4.1.2,No,-
22CS999924,Information saknas,01,489423,48.35,51.61,FALSE,B.1.617.2,4.1.2,VOC,-
22CS999925,Information saknas,01,489423,2.45,99.12,TRUE,AY.9.2,4.1.2,No,A222V
22CS999926,Information saknas,01,489423,2.48,99.4,TRUE,AY.9.2,4.1.2,No,A222V;D614G;P681R
22CS999927,Information saknas,01,489423,4.12,98.6,TRUE,AY.9.2,4.1.2,No,G142D;A222V
22CS999928,Information saknas,01,489423,10.1,93.0,TRUE,AY.9.2,4.1.2,No,G142D;A222V
22CS999929,Information saknas,01,489423,4.19,95.81,TRUE,AY.9.2,4.1.2,No,A222V
22CS999930,Information saknas,01,489423,4.86,95.94,TRUE,AY.4,4.1.2,No,T95I
22CS999931,Information saknas,01,489423,2.93,98.67,TRUE,AY.122,4.1.2,No,-
22CS999932,Information saknas,01,489423,4.19,98.81,TRUE,AY.9.2,4.1.2,No,A222V;D614G;P681R
22CS999933,Information saknas,01,489423,31.14,75.53,FALSE,B.1.617.2,4.1.2,VOC,G142D;L452R;D614G;P681R
22CS999934,Information saknas,01,489423,4.19,98.19,TRUE,AY.9.2,4.1.2,No,A222V;Q613H;D614G
22CS999935,Information saknas,01,489423,6.24,94.73,TRUE,AY.42,4.1.2,No,-
22CS999936,Information saknas,01,489423,5.28,96.62,TRUE,AY.122,4.1.2,No,D614G
22CS999937,Information saknas,01,489423,3.64,97.2,TRUE,B.1.617.2,4.1.2,VOC,-
22CS999938,Information saknas,01,489423,7.24,95.33,TRUE,AY.9.2,4.1.2,No,A222V;D614G;P681R
22CS999939,Information saknas,01,489423,3.57,98.47,TRUE,AY.9.2,4.1.2,No,-
22CS999940,Information saknas,01,489423,5.67,95.29,TRUE,AY.43,4.1.2,No,-
22CS999941,Information saknas,01,489423,11.17,96.02,TRUE,AY.9.2,4.1.2,No,A222V;D614G;P681R
22CS999942,Information saknas,01,489423,9.15,97.38,TRUE,AY.122,4.1.2,No,D614G;P681R
22CS999943,Information saknas,01,489423,99.8,1.91,FALSE,Unassigned,4.1.2,No,-
22CS999944,Information saknas,01,489423,11.97,93.91,TRUE,AY.9.2,4.1.2,No,A222V;D614G;P681R
22CS999945,Information saknas,01,489423,8.4,94.74,TRUE,AY.9,4.1.2,No,A222V;D614G;P681R
22CS999946,Information saknas,01,489423,19.96,84.07,FALSE,B.1.617.2,4.1.2,VOC,L452R;D614G;P681R
22CS999947,Information saknas,01,489423,58.11,49.25,FALSE,B.1.617.2,4.1.2,VOC,G142D;A222V;L452R;T478K;D614G;P681R
22CS999948,Information saknas,01,489423,77.83,22.04,FALSE,B.1.617.2,4.1.2,VOC,L452R;T478K;D614G;P681R
```

**Comment by henningonsbring** [comment]
Files created by the analysis:
```
ll truecoyote/results/
total 4672
-rw-rw----+   2 hiseq.clinical users    8122 Jul 28 14:33 489423_2022-07-28_pangolin_classification_format4.txt
-rw-rw----+   2 hiseq.clinical users 1461655 Jul 28 14:33 489423.consensus.fa
-rw-rw----+   2 hiseq.clinical users    1130 Jul 28 14:33 489423_komplettering.csv
lrwxrwxrwx    1 hiseq.clinical users     151 Jul 28 14:33 489423_multiqc.html -> /home/proj/stage/mutant/cases/truecoyote/results/QCStats/articNcovNanopore_sequenceAnalysisMedaka_multiqcNanopore/truecoyote_220728-141910_multiqc.html
lrwxrwxrwx    1 hiseq.clinical users     169 Jul 28 14:33 489423_multiqc.json -> /home/proj/stage/mutant/cases/truecoyote/results/QCStats/articNcovNanopore_sequenceAnalysisMedaka_multiqcNanopore/truecoyote_220728-141910_multiqc_data/multiqc_data.json
-rw-rw----+   2 hiseq.clinical users   10827 Jul 28 14:33 489423.pangolin.csv
lrwxrwxrwx    1 hiseq.clinical users      80 Jul 28 14:33 489423.qc.csv -> /home/proj/stage/mutant/cases/truecoyote/results/truecoyote_220728-141910.qc.csv
lrwxrwxrwx    1 hiseq.clinical users      92 Jul 28 14:33 489423.typing_summary.csv -> /home/proj/stage/mutant/cases/truecoyote/results/truecoyote_220728-141910.typing_summary.csv
lrwxrwxrwx    1 hiseq.clinical users      93 Jul 28 14:33 489423.variant_summary.csv -> /home/proj/stage/mutant/cases/truecoyote/results/truecoyote_220728-141910.variant_summary.csv
drwxrws---+   5 hiseq.clinical users    4096 Jul 28 14:30 articNcovNanopore_Genotyping_typeVariants
drwxrws---+   2 hiseq.clinical users    4096 Jul 28 14:22 articNcovNanopore_sequenceAnalysisMedaka_articDownloadScheme
drwxrws---+   2 hiseq.clinical users    4096 Jul 28 14:26 articNcovNanopore_sequenceAnalysisMedaka_articGuppyPlex
drwxrws---+   2 hiseq.clinical users  131072 Jul 28 14:33 articNcovNanopore_sequenceAnalysisMedaka_articMinIONMedaka
drwxrws---+   2 hiseq.clinical users    4096 Jul 28 14:33 articNcovNanopore_sequenceAnalysisMedaka_pangolinTyping
drwxrws---+   2 hiseq.clinical users    4096 Jul 28 14:22 articNcovNanopore_sequenceAnalysisMedaka_versions
-rw-rw----+   2 hiseq.clinical users      95 Jul 28 14:33 instrument.properties
-rw-rw----+   2 hiseq.clinical users  491261 Jul 28 14:33 nextflow.log
drwxrws---+   2 hiseq.clinical users    4096 Jul 28 14:33 pipeline_info
drwxrws---+   3 hiseq.clinical users    4096 Jul 28 14:32 qc_pass_climb_upload
drwxrws---+   2 hiseq.clinical users    4096 Jul 28 14:32 qc_plots
drwxrws---+   4 hiseq.clinical users    4096 Jul 28 14:26 QCStats
-rw-rw----+   2 hiseq.clinical users    4591 Jul 28 14:33 sars-cov-2_489423_results.csv
-rw-rw----+   2 hiseq.clinical users   19288 Jul 28 14:33 sars-cov-2_489423_variants.csv
-rw-rw----+   1 hiseq.clinical users    4306 Jul 28 14:33 trailblazer_config.yaml
-rw-rw----+   1 hiseq.clinical users    8560 Jul 28 14:32 truecoyote_220728-141910.qc.csv
-rw-rw----+   1 hiseq.clinical users    8813 Jul 28 14:32 truecoyote_220728-141910.typing_summary.csv
-rw-rw----+   1 hiseq.clinical users   46707 Jul 28 14:32 truecoyote_220728-141910.variant_summary.csv
-rw-rw----+   1 hiseq.clinical users   34002 Jul 28 14:33 truecoyote_deliverables.yaml
drwxrws---+ 209 hiseq.clinical users   32768 Jul 28 14:32 work
```

**Comment by henningonsbring** [comment]
Test on stage and dry-run indicate that FOHM upload is working. When running `cg upload fohm preprocess-all --dry-run -c truecoyote`:

```
...
Fetching files from version 103243
Would have copied /home/proj/stage/housekeeper-bundles/truecoyote/2022-07-28/01_SE100_22CS999945_1.fastq.gz to /home/proj/stage/mutant/fohm/2022-07-28/rawdata
Would have copied /home/proj/stage/housekeeper-bundles/truecoyote/2022-07-28/01_SE100_22CS999945.vcf to /home/proj/stage/mutant/fohm/2022-07-28/rawdata
Would have copied /home/proj/stage/housekeeper-bundles/truecoyote/2022-07-28/01_SE100_22CS999945.consensus.fasta to /home/proj/stage/mutant/fohm/2022-07-28/rawdata
|████████████████████████████████████████| 35/35 [100%] in 0.4s (77.78/s)
Connected (version 2.0, client OpenSSH_8.2p1)
Authentication (publickey) successful!
[chan 0] Opened sftp connection (server version 3)
Sending /home/proj/stage/mutant/fohm/2022-07-28/rawdata/01_SE100_2022-07-28_pangolin_classification_format4.txt via SFTP, dry-run True
|████████████████████████████████████████| 1/1 [100%] in 0.1s (9.93/s)
[chan 0] sftp session closed.
Sending 01_SE100_2022-07-28_komplettering.csv report file to mariaropat@gmail.com,maryia.ropart@scilifelab.se, dry run: True
Upload to FOHM completed
```

**Comment by henningonsbring** [comment]
Illumina analysis of ablequagga ran as expected with result file:
```
Sample,Selection,Region Code,Ticket,%N_bases,%10X_coverage,QC_pass,Lineage,Pangolin_data_version,VOC,Mutations
22CS104078,Allmän övervakning öppenvård,01,373734,23.62,76.28,FALSE,Unassigned,PUSHER-v1.12,No,G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;Q954H;N969K
22CS104064,Allmän övervakning slutenvård,01,373734,47.82,52.14,FALSE,Unassigned,PUSHER-v1.12,No,G339D;D614G;N969K
22CS104069,Allmän övervakning öppenvård,01,373734,3.31,96.54,TRUE,BA.2,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104076,Allmän övervakning öppenvård,01,373734,4.92,95.01,TRUE,BA.2.12.1,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;L452Q;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104052,Allmän övervakning slutenvård,01,373734,7.09,92.76,TRUE,BA.2,PUSHER-v1.12,No,G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104054,Allmän övervakning slutenvård,01,373734,5.27,94.58,TRUE,BA.2.71,PANGO-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104060,Allmän övervakning slutenvård,01,373734,2.99,96.85,TRUE,BA.2.23,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;K417N;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104066,Allmän övervakning öppenvård,01,373734,4.79,95.06,TRUE,BA.2,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;K417N;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104053,Allmän övervakning slutenvård,01,373734,0.21,99.63,TRUE,BA.5.1,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;K417N;N440K;L452R;S477N;T478K;E484A;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104063,Allmän övervakning slutenvård,01,373734,0.28,99.57,TRUE,BA.2.71,PANGO-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104059,Allmän övervakning slutenvård,01,373734,0.65,99.21,TRUE,BA.2.3,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104075,Allmän övervakning öppenvård,01,373734,0.29,99.56,TRUE,BA.2,PANGO-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104072,Allmän övervakning öppenvård,01,373734,0.30,99.57,TRUE,BA.2.9,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;K417N;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104079,Allmän övervakning öppenvård,01,373734,9.85,90.11,TRUE,BA.2.13,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;K417N;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;N764K;D796Y;Q954H;N969K
22CS104055,Allmän övervakning slutenvård,01,373734,0.13,99.74,TRUE,BA.2,PUSHER-v1.12,No,G142D;A222V;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104074,Allmän övervakning öppenvård,01,373734,0.12,99.75,TRUE,BA.2.12.1,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;L452Q;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104068,Allmän övervakning öppenvård,01,373734,0.51,99.36,TRUE,BA.2.12.1,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;K417N;N440K;L452Q;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104061,Allmän övervakning slutenvård,01,373734,0.15,99.72,TRUE,BA.2,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;K417N;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104070,Allmän övervakning öppenvård,01,373734,0.14,99.73,TRUE,BA.2.12.1,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;L452Q;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104077,Allmän övervakning öppenvård,01,373734,0.12,99.75,TRUE,BA.2,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104056,Allmän övervakning slutenvård,01,373734,0.12,99.75,TRUE,BA.2.9,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104062,Allmän övervakning slutenvård,01,373734,0.13,99.74,TRUE,BA.2.12.1,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;L452Q;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104065,Allmän övervakning slutenvård,01,373734,0.09,99.75,TRUE,BA.4.1.2,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;L452R;S477N;T478K;E484A;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104057,Allmän övervakning slutenvård,01,373734,0.08,99.80,TRUE,BA.2,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
22CS104058,Allmän övervakning slutenvård,01,373734,0.07,99.80,TRUE,BA.2,PUSHER-v1.12,No,G142D;G339D;S373P;S375F;N440K;S477N;T478K;E484A;Q493R;Q498R;N501Y;Y505H;D614G;H655Y;N679K;P681H;N764K;D796Y;Q954H;N969K
Negk,Allmän övervakning,01,373734,99.88,0.12,FALSE,Unassigned,PUSHER-v1.12,No,-
22CS104067,Allmän övervakning öppenvård,01,373734,95.19,4.80,FALSE,Unassigned,PUSHER-v1.12,No,-
22CS104071,Allmän övervakning öppenvård,01,373734,99.78,0.22,FALSE,Unassigned,PUSHER-v1.12,No,-
22CS104073,Allmän övervakning öppenvård,01,373734,87.60,12.40,FALSE,Unassigned,PUSHER-v1.12,No,-
```

Upload to FOHM in dry-run worked as well, `cg upload fohm preprocess-all --dry-run -c ablequagga` gave output:
```
Would have copied /home/proj/stage/housekeeper-bundles/ablequagga/2022-07-28/01_SE100_22CS104077.consensus.fasta to /home/proj/stage/mutant/fohm/2022-07-29/rawdata
Fetch latest version from bundle ablequagga
Fetching files with tags in [ACC9971A28]
Fetching files from version 103244
Would have copied /home/proj/stage/housekeeper-bundles/ablequagga/2022-07-28/01_SE100_22CS104079_1.fastq.gz to /home/proj/stage/mutant/fohm/2022-07-29/rawdata
Would have copied /home/proj/stage/housekeeper-bundles/ablequagga/2022-07-28/01_SE100_22CS104079_2.fastq.gz to /home/proj/stage/mutant/fohm/2022-07-29/rawdata
Would have copied /home/proj/stage/housekeeper-bundles/ablequagga/2022-07-28/01_SE100_22CS104079.vcf to /home/proj/stage/mutant/fohm/2022-07-29/rawdata
Would have copied /home/proj/stage/housekeeper-bundles/ablequagga/2022-07-28/01_SE100_22CS104079.consensus.fasta to /home/proj/stage/mutant/fohm/2022-07-29/rawdata
|████████████████████████████████████████| 23/23 [100%] in 0.3s (74.35/s)
Connected (version 2.0, client OpenSSH_8.2p1)
Authentication (publickey) successful!
```

File: mutant/modules/delivery.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+    def map_sampleid_to_barcode(self) -> dict:
+        sampleid_to_barcode = {}
+        for barcode in self.barcode_to_sampleid:
+            sampleid = self.barcode_to_sampleid[barcode]
+            sampleid_to_barcode[sampleid] = barcode
+        return sampleid_to_barcode
+
+    def translate_basename_to_nanopore_format(self, sampleid_to_barcode: dict) -> dict:
+        illunmina_to_nanopore_base = {}
```
Comment (sofstam) [inline mutant/modules/delivery.py:41]: Maybe `illumina_to_nanopore_base` instead?

Comment (henningonsbring) [reply mutant/modules/delivery.py:41]: should I name the function `translate_ullumina_to_nanopore_base`?

Comment (sofstam) [reply mutant/modules/delivery.py:41]: Just a typo in `illunmina_to_nanopore_base = {} `

Comment (henningonsbring) [reply mutant/modules/delivery.py:41]: aaah! now I see it :-)

**Pull request merged**